c2/c2-client-bundle/c2-client-service/pom.xml

```diff
@@ -43,5 +43,10 @@ limitations under the License.
             <artifactId>c2-client-http</artifactId>
             <version>1.18.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.21</version>
```

Comment:
Please externalise the version to the main nifi pom so everything is in one place.

Reply:
I checked the occurrences of this dependency in other modules. Everywhere it is used with version. To not break consistency I would leave this as it is, if you don't mind